mesonbuild/interpreter.py

```diff
@@ -2174,6 +2174,10 @@ def parse_default_options(self, default_options):
                             if not (self.environment.had_argument_for(option) or
                                     any([k.startswith(option + '=') for k in default_options])):
                                 self.coredata.set_builtin_option(option, coredata.get_builtin_option_default(option, value))
+            elif self.coredata.is_backend_option(key):
+                if self.subproject != '':
+                    continue # Only the master project is allowed to set global options.
+                self.coredata.set_backend_option(key, value)
```

Comment:
Note that PR #3528 already fix this.

Reply:
Great, then I can rework my PR when your PR is merged!

Reply:
The first part was merged. Is the correct time for this now?